CONTRIBUTING.rst

```diff
@@ -2,42 +2,70 @@
 Contributing
 ============
 
-.. contents:: :local:
+All projects under the Pylons Projects, including this one, follow the
+guidelines established at [How to
+Contribute](http://www.pylonsproject.org/community/how-to-contribute) and
+[Coding Style and
+Standards](http://docs.pylonsproject.org/en/latest/community/codestyle.html).
+
+You can contribute to this project in several ways.
+
+* [File an Issue on GitHub](https://github.com/Pylons/pyramid/issues)
+* Fork this project and create a branch with your suggested change. When ready,
+  submit a pull request for consideration. [GitHub
+  Flow](https://guides.github.com/introduction/flow/index.html) describes the
+  workflow process and why it's a good practice. When submitting a pull
+  request, sign
+  [CONTRIBUTORS.txt](https://github.com/Pylons/deform/blob/master/CONTRIBUTORS.txt)
+  if you have not yet done so.
+* Join the IRC channel #pyramid on irc.freenode.net.
 
-Discuss first
--------------
 
-For changes open an issue p on Github for discussion preferably before starting any coding. Ask the authors opinion how the issue should be approached as this will make it easier to merge the pull request later.
+Discuss proposed changes first
+------------------------------
+
+To propose a change, open an issue on Github for discussion, preferably before
+starting any coding. Ask the authors' opinion how the issue should be
+approached, as this will make it easier to merge the pull request later.
+
 
 Unit tests
 ----------
 
-All features must be covered by unit tests, so that test coverage stays in 100%.
+All features must be covered by unit tests, so that test coverage stays at
+100%.
 
 To run tests::
 
     pip install tox  # System wide installation
     tox
 
-This will run tests for Python 2.x, Python 3.x, PyPy, functional, coverage, documentation.
+This will run tests for Python 2.x, Python 3.x, PyPy, functional, coverage,
+and documentation.
+
 
 Functional tests
 ----------------
 
 All features must be covered by functional tests and have example use.
 
+
 Preparing compatible browser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Functional tests run on Firefox 43 and Selenium 2.56. Below are instructions for OSX:
+Functional tests run on Firefox 43 and Selenium 2.56. Below are instructions
```

Comment:
Would you open a new issue for this? @miohtama would know the minimum requirements. I merely renamed the file and added some spit and polish. Thank you!